src/components/UserDetailPage/UserDetailPage.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Author } from '../../models/Author';
+import { Book } from '../../models/Book';
+import { Category } from '../../models/Category';
+import BookCard from '../BookCard/BookCard';
+
+function UserDetailPage() {
+
+  const [books, setBooks] = useState<Book[]>([]);
+
+  useEffect(() => {
+    fetch('https://localhost:8000/my/favorites')
+      .then(response => response.json())
+      .then(data => {
+        const res = data.data.map((book: Book) => {
```

Comment:
it does not work at my side. I think it is  data.favorites.map(....
- as per endpoint return value:
return $this->json(['favorites' => $user->getFavorites()]);

Reply:
Also the token for authorization needs to be passed, too